sdk/policy/arm-policy/samples-dev/policyAssignmentsCreateByIdSample.ts

```diff
@@ -18,7 +21,9 @@ import { DefaultAzureCredential } from "@azure/identity";
  * x-ms-original-file: specification/resources/resource-manager/Microsoft.Authorization/stable/2021-06-01/examples/createPolicyAssignmentById.json
  */
 async function createOrUpdatePolicyAssignmentById() {
-  const subscriptionId = "00000000-0000-0000-0000-000000000000";
+  const subscriptionId =
```

Comment:
Can you check if subscriptionId is required for these operation ? also checked the swagger example https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resources/resource-manager/Microsoft.Authorization/stable/2021-06-01/examples/createPolicyAssignmentById.json, there is no subscriptionId parameter. Maybe we should fix it in the codegen side, would you mind create an issue to track this ? Thanks

Reply:
https://github.com/Azure/autorest.typescript/issues/1864